tests/csm/rest/test_capacity_quota.py

```diff
@@ -502,27 +538,23 @@ def test_40639(self):
         test_case_name = cortxlogging.get_frame()
         self.log.info("##### Test started -  %s #####", test_case_name)
         test_cfg = self.csm_conf["test_40635"]
-        quota_type = test_cfg["quota_type"]
         enabled = test_cfg["enabled"]
         max_size = test_cfg["max_size"]
         max_objects = test_cfg["max_objects"]
-        self.log.info("Step 2: Create bucket under above IAM user")
-        self.log.info("Verify Create bucket: %s with access key: %s and secret key: %s",
-                      self.bucket, self.akey, self.skey)
-        bucket_created = s3_misc.create_bucket(self.bucket, self.akey, self.skey)
-        assert bucket_created, "Failed to create bucket"
         self.log.info("Step 2: Start Multipart upload S3 operations of X Mb")
+        bucket = "iam-user-bucket-" + str(int(time.time_ns()))
+        obj_name = f'{self.obj_name_prefix}{time.perf_counter_ns()}'
         res = self.create_bucket_to_upload_parts(
-            self.bucket,
-            self.obj_name,
-            max_size,
+            bucket,
+            obj_name,
+            int(max_size/1024),
```

Comment:
Can u help me understand what is the max_size unit assumed throught out these tests and libs?

Reply:
max size considered is in bytes and block size of 1K or 1M is considered as per test requirement.